README.md

```diff
@@ -102,7 +96,17 @@ See the full example of action step definition (in example are used non-default
         "projects-title-filter": ["Community Outreach Initiatives", "CDD Project"] 
       }
     ]'
+    
+    # output directory path for generated documentation
     output-path: "/output/directory/path"
+
+    # project state mining feature de/activation
+    project-state-mining: true
+    
+    # project verbose (debug) logging feature de/activation
+    verbose-logging: true
+
+    
```

Comment:
```suggestion
```

Reply:
Typo fix in commit: [ab098bf](https://github.com/AbsaOSS/living-doc-generator/pull/27/commits/ab098bf257d34323b0d13adef54e20a23ec902eb)